### PR TITLE
Rename Kind  demo cluster + small improvements

### PR DESCRIPTION
**Pull request by robertwol**
- rebrand Kind cluster from aws-demo to ocm-demo
- shorten the interval waiting time to resource reconciliation.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -4,4 +4,5 @@ run:
 	@./00-setup-demo/setup.sh --offline-mode 1
 
 teardown:
-	kind delete cluster --name aws-demo
+	kind delete cluster --name ocm-demo
+	rm -r 00-setup-demo/charts/telepresence/
```
Comment (phoban01) [inline Makefile:8]: this has been removed from the repo so is not necessary to add to makefile.
```suggestion
```

**Review by phoban01 — COMMENTED** [review]
I don't think it's necessary to reduce the reconcile intervals; in fact it will not make much difference because we have setup a webhook that will trigger an immediate reconciliation of resources.

File: Makefile (hunk 1 above)
Comment (robertwol) [reply Makefile:8]: it is actually pulled here https://github.com/open-component-model/demo-secure-delivery/blob/main/00-setup-demo/lib.sh#L325 and is not removed after cleanup

**Comment by robertwol** [comment]
> 

@phoban01 
I am not sure if the webhook works instantly then. 
1st deployment after the v1.0.0 release is created works almost instantly.
but the second release takes ages until the release got deployed.
it could be the ocm controller interval or something. can you advise how to improve?

File: Makefile (hunk 1 above)
Comment (phoban01) [reply Makefile:8]: then we should remove the pull I would suggest.

Comment (robertwol) [reply Makefile:8]:
```
    if [ ! -d $CHART_DIR/telepresence ]; then
        helm repo add datawire https://app.getambassador.io || true
        helm pull datawire/telepresence --untar=true --untardir=$CHART_DIR
    fi
```

I guess the nginx-ingress is also not needed 

Seems only Gitea is installed with Helm
```
helm ls -A                                             
NAME    NAMESPACE       REVISION        UPDATED                                 STATUS          CHART           APP VERSION
gitea   gitea           1               2023-07-27 12:03:49.610741 +0200 CEST   deployed        gitea-8.3.0     1.19.3 
```

Comment (robertwol) [reply Makefile:8]: I think nginx-ingress is intalled via the manifest here:
https://github.com/open-component-model/demo-secure-delivery/blob/main/00-setup-demo/lib.sh#L77-L83

**Pull request merged**